### DISCUSSION TITLE
Appreciate the constant support

### DISCUSSION THREAD

**Discussion by ShaDisNX255** [adil192/SamsungAppsPatcher#16] [General]
Just wanted to say I really do appreciate the support you've given this repo even if you don't have a Samsung device anymore. I've been meaning to try and patch the newer apps myself and report back on any issue I find, but been busy lately.

I will hopefully try it in the next day or 2. I've got it mostly set-up, I think last time I tried I got an error related to missing a way to sign the app which I'm sure your instructions do give the option to set something up to sign the apk

Anyway just wanted to give you my thanks. I know it may not mean much but I really am grateful.

**Comment by adil192** [comment]
Glad I could help! :)

**Reply by HamadaMuc** [reply]
That's really nice of you.  I wish you could keep some apps updated.  Like Samsung Health .
